templates/api-server-deployment.yaml

```diff
@@ -69,12 +69,18 @@ spec:
               value: {{ .Values.stats.host }}.{{ default .Release.Namespace .Values.global.monitoring.namespace }}
             - name: STATS_PORT
               value: {{ .Values.stats.port  | quote }}
+            - name: STATS_TAGS
+              value: "host:$(K8S_POD_NAME)"
             - name: STATS_PROTOCOL
               value: {{ .Values.stats.protocol }}
             - name: STATS_TELEGRAF
               value: "1"
             - name: STATS_SAMPLE_RATE
               value:  {{ .Values.stats.sampleRate | quote }}
+            - name: K8S_POD_NAME
```

Comment:
the env K8S_POD_NAME must appear above another env that wants to interpolate that value, so it must come before STATS_TAGS.  This applies to each of these files where it appears.